spec/helpers/block.js

```diff
@@ -37,7 +37,7 @@ describe('partial and block helpers', function () {
 
     it('should not trigger an error if block name is empty', function (done) {
         var templates = {
-            template: '{{#partial "page" "2134"}}World{{/partial}}{{> layout}}',
+            template: '{{#partial "page"}}World{{/partial}}{{> layout}}',
```

Comment:
Why did this test need to change?

Reply:
test needed to change as more than one argument is being passed to partial helper.  We do not have access to options in helper if that is the case.